### PR TITLE
Find Items feature

### LINKED ISSUE #36: As a user, I want to be able to find items by using a search string

### PR DESCRIPTION
**Pull request by Rahul0506**
By definition, keywords cannot be multi-word between them currently. Ex: "foo bar" is 2 keywords

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FindItemCommand.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * Keyword matching is case insensitive.
+ */
+public class FindItemCommand extends Command {
+
+    public static final String COMMAND_WORD = "find";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all items whose names contain any of "
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
+            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
+            + "Example: " + COMMAND_WORD + " alice bob charlie";
```
Comment (justacasul) [inline src/main/java/seedu/address/logic/commands/FindItemCommand.java:20]: Will there be an item named alice bob charlie?

**Review by justacasul — CHANGES_REQUESTED** [review]
Please remember to rebase

File: src/main/java/seedu/address/logic/commands/FindItemCommand.java (hunk 1 above)
Comment (Rahul0506) [reply src/main/java/seedu/address/logic/commands/FindItemCommand.java:20]: changed to "apple banana carrot"

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 2]
```diff
@@ -9,6 +9,9 @@
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
+    public static final String MESSAGE_ITEMS_LISTED_OVERVIEW = "%1$d items listed!";
     public static final String MESSAGE_NO_ITEM_FOUND = "No such item '%1$s' found, did you type the name correctly?";
+    public static final String MESSAGE_NO_ITEM_MATCH = "No items match the search string. "
+            + "Try again with a different input.";
```
Comment (aqcd) [inline src/main/java/seedu/address/commons/core/Messages.java]: Just "No items match the search string." should suffice here

**Review by aqcd — APPROVED** [review]
LGTM

**Pull request merged**